### PR TITLE
Azspice kgo update

### PR DESCRIPTION
**Pull request by james-bruten-mo**
Update the kgo update script to run on azspice. This uses sudo commands with umadmin, instead of ssh'ing to frum.
Have tested by installing the kgo for UM:#7572 on azspice.

### CODE REVIEW

**Comment by james-bruten-mo** [comment]
Assinging Erica as UM:#7572 code reviewer and Cameron so he can have a look how these scripts work

**Comment by james-bruten-mo** [comment]
Also modifying the EX platforms, so kgo is installed on EXAB and rsynced to the EXZ. Will require a further PR to rsync to C/D when they are available

File: kgo_updates/meto_update_kgo.sh [hunk 1]
```diff
@@ -40,15 +53,15 @@ fi
 
 # Prompt user for Update Details
 echo "Enter the platforms requiring a kgo update"
-echo "Enter platforms lowercase and space separated, eg. spice xc40 ex1a"
+echo "Enter platforms lowercase and space separated, eg. spice xc40 ex1a azspice"
 read platforms
-if [[ $platforms == *"spice"* ]] || [[ $platforms == *"xc40"* ]]; then
+if [[ $platforms == "spice"* ]] || [[ $platforms == *"xc40"* ]]; then
```
Comment (ericaneininger) [inline kgo_updates/meto_update_kgo.sh:58]: Same comment regarding removing the preceding `*` - you are forcing the requirement that spice is first in the list or else the spice test will always fail.

**Review by ericaneininger — APPROVED** [review]
Thank you for indulging pedantic comments and making suggested changes.  happy to approve now.

**Review by cameronbateman-mo — APPROVED** [review]
All seems good, but my bash knowledge is low so beware xD

**Pull request merged**